packages/adminpanel-v2/src/stores/CustomerStore.ts

```diff
@@ -150,10 +150,12 @@ function createCustomers() {
 		setCustomers(customers);
 	};
 
+	const getNewDateString = () => {return new Date().toLocaleString('en-GB', { timeZone: 'UTC' }).split(',')[0]}
```

Comment:
Nice! Here you can even remove the curly brackets if you want to be super smooth and do `getNewDateString = () => (...)`